### PR TITLE
Gauntlet recommendation for MAI / MIMATIC deprecation phase 2

### PR DESCRIPTION
**Pull request by jbatch01**

### CODE REVIEW

File: src/20240122_AaveV3Optimism_GauntletRecommendationForMAIMIMATICDeprecationPhase2/GauntletRecommendationForMAIMIMATICDeprecationPhase2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Gauntlet recommendation for MAI / MIMATIC deprecation phase 2"
+author: "Gauntlet"
+discussions: "https://governance.aave.com/t/arfc-gauntlet-recommendation-for-mai-mimatic-deprecation-phase-2/15957"
+---
+
+## Simple Summary
+
+A proposal to make parameter changes on Aave Optimism v3. For more details, see the full forum post [here](https://governance.aave.com/t/arfc-gauntlet-recommendation-for-mai-mimatic-deprecation-phase-2/15957).
```
Comment (brotherlymite) [inline src/20240122_AaveV3Optimism_GauntletRecommendationForMAIMIMATICDeprecationPhase2/GauntletRecommendationForMAIMIMATICDeprecationPhase2.md:9]:
```suggestion
A proposal for MAI / MIMATIC deprecation on Aave Optimism v3. For more details, see the full forum post [here](https://governance.aave.com/t/arfc-gauntlet-recommendation-for-mai-mimatic-deprecation-phase-2/15957).
```

File: src/20240122_AaveV3Optimism_GauntletRecommendationForMAIMIMATICDeprecationPhase2/GauntletRecommendationForMAIMIMATICDeprecationPhase2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Gauntlet recommendation for MAI / MIMATIC deprecation phase 2"
+author: "Gauntlet"
+discussions: "https://governance.aave.com/t/arfc-gauntlet-recommendation-for-mai-mimatic-deprecation-phase-2/15957"
```
Comment (eboadom) [inline src/20240122_AaveV3Optimism_GauntletRecommendationForMAIMIMATICDeprecationPhase2/GauntletRecommendationForMAIMIMATICDeprecationPhase2.md:4]:
```suggestion
title: "Gauntlet recommendation for MAI / MIMATIC deprecation phase 2"
author: "Gauntlet"
discussions: "https://governance.aave.com/t/arfc-gauntlet-recommendation-for-mai-mimatic-deprecation-phase-2/15957"
snapshot: "https://snapshot.org/#/aave.eth/proposal/0x67a6941140c0c0662cfbf99254100f58930afb6763b8040c4bdbd0dfbb2a952b"
```

**Review by eboadom — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Comment by jbatch01** [comment]
@brotherlymite @eboadom - Running the test locally, I saw that the Etherscan Polygon URL changed, which I believe was causing the above `Test error`

**Comment by brotherlymite** [comment]
> @brotherlymite @eboadom - Running the test locally, I saw that the Etherscan Optimism URL changed, which I believe was causing the above `Test error`

The error on the CI is because of the rpc, so the error `missing trie node ...` could be ignored as it is passing locally.

**Review by brotherlymite — APPROVED** [review]

**Review by eboadom — APPROVED** [review]

**Pull request merged**